### DISCUSSION TITLE
Persistent tooltip with interactivity

### DISCUSSION THREAD

**Discussion by thewocky** [chakra-ui/chakra-ui#5816] [Ideas]
Hello,

I'd like to implement a tooltip with a link. Whereby the tooltip appears on hover like usual, but persists so that you can click (or otherwise interact with) a node within the tooltip. Should look something like this:
```
<Tooltip label={<span>Check out <a href="https://github.com/chakra-ui/chakra-ui/">chakra-ui</a></span>} defaultIsOpen persistUntilClickOutside>
  <button style={{ fontSize: 25, pointerEvents: "all" }}>
    Click the link in the tooltip
  </button>
</Tooltip>
```
Has this functionality been requested or tried before, and if so were there blockers that prevented it from making it in? Otherwise I'd like to work on it and contribute.

**Comment by alabhyajindal** [comment]
Did you find any solution for this? I am trying to do the exact same thing right now.

**Comment by linghaolin** [comment]
I found a work around. We can set `closeDelay={2000}` to delay closing the tooltip so users can interact with things in the tooltip. Not perfect, but it's good enough.

**Comment by tirkarthi** [comment]
> Whether the tooltip's content is interactive. In this mode, the tooltip will remain open when user hovers over the content.

In chakra 3 this is possible with `interactive` prop.

https://www.chakra-ui.com/docs/components/tooltip
